CHANGES.md

```diff
@@ -3,7 +3,11 @@ Change Log
 
 ### 1.11 - 2015-07-01
 
+* Breaking changes
+  * Removed `Camera.clone` that was deprecated in 1.10.
 * Deprecated
+  * `CatmullRomSpline` has been deprecated and will be removed in Cesium 1.12.
```

Comment:
@GatorScott I added cubic b-splines and bezier splines a while back for model animation. They're sitting in the `splines` branch if we ever need them.